swagger.go

```diff
@@ -76,7 +77,12 @@ func CustomWrapHandler(config *Config, h *webdav.Handler) gin.HandlerFunc {
 			DefaultModelsExpandDepth int
 		}
 
-		var matches []string
+		matches := rexp.FindStringSubmatch(c.Request.RequestURI)
+
+		if len(matches) == 0 {
+			c.Request.RequestURI = fmt.Sprintf("%vindex.html", c.Request.RequestURI)
+		}
+
 		if matches = rexp.FindStringSubmatch(c.Request.RequestURI); len(matches) != 3 {
```

Comment:
This is funny.
Is there any special reason why are u calling `rexp.FindStringSubmatch(c.Request.RequestURI)` two times?